docs/api-types.md

```diff
@@ -0,0 +1,100 @@
+# API types documentation
+
+This document describes format of objects used in various API's across the project.
+For other details, see swagger documentation of [educator](/specs/disciplina/educator/api/educator.yaml), [student](/specs/disciplina/educator/api/student.yaml) and [witness](/specs/disciplina/witness/api/witness.yaml) API.
+
+## Common objects
+
+Some types of values are always represented in the same format, among them:
```

Comment:
`s/among them/for example/` maybe?

Reply:
I meant to say that I enlist all such types. Maybe `they are`?

Reply:
`Among them` really says quite the opposite. You should say `The following types are always represented in the same format` then.